client/main.lua

```diff
@@ -2,6 +2,11 @@ local cam = nil
 local charPed = nil
 local QBCore = exports['qb-core']:GetCoreObject()
 
+local clothingScript = "qb-clothing"
+if GetResourceState("fivem-appearance") == "started" then
```

Comment:
This is no longer called fivem-appearance

Reply:
Actually fivem-appearance is the old resource but people still use it, illenium-appearance is just the new one.

Reply:
Yeah, I'm unsure if support for illenium-appearance is even the same as support for fivem-appearance.
So changing it from fivem-appearance to illenium-appearance without looking over the rest of the code would perhaps be an bad idea.